src/controllers/checkins.ts

```diff
@@ -67,6 +67,41 @@ export const create_event: Express.RequestHandler = async (req, res) => {
   res.json(id);
 }
 
+export const update_one_event: Express.RequestHandler = async (req, res) => {
+  const event_update: EventUpdate = req.body;
+
+  const sheet = google.sheets("v4");
+
+  const read_result = await sheet.spreadsheets.values.get({
+      spreadsheetId: process.env.SHEET_ID,
+      auth: sheet_auth(),
+      range: `${EVENT_SHEET_ID}!A1:D`
+  });
+
+  const rows = read_result.data.values as string[][];
+  const ser = serialize_rows(rows) as Array<Event>;
+
+  const eventIndex = ser.findIndex((r) => event_update.old_alias === r.alias);
+  let mut_event = ser[eventIndex];
+  const sheetIndex = eventIndex + 2;
+
+  mut_event.alias = event_update.new_alias;
+
+  const row = [...Object.values(mut_event)];
+
+  const request = {
+      spreadsheetId: process.env.SHEET_ID,
+      range: "Event!A" + sheetIndex + ":D" + sheetIndex,
+      valueInputOption: "RAW",
+      auth: sheet_auth(),
+      resource: {
+          values: [row]
+      }
+  }
+
+  sheet.spreadsheets.values.update(request);
```

Comment:
Please send back a response. Otherwise, this request will hang forever. `res.json(something goes in here)`